tests/checkpoint_chunks/run.sh

```diff
@@ -33,7 +33,7 @@ done
 
 # Set the failpoint to kill the lightning instance as soon as one chunk is imported
```

Comment:
fix comment

Reply:
in fact it fails while kvs > xxx

Reply:
How about this? 

```sh
# Set the failpoint to kill the lightning instance as soon as 
# one file (after writing totally $ROW_COUNT rows) is imported
```

Since one chunk contains `$ROW_COUNT`, the effect is the same. The `>=` ensures we don't accidentally break in the middle of the file which increases the retry count.

Reply:
ok